.github/workflows/build-library.yml

```diff
@@ -47,4 +47,13 @@ jobs:
           python -m pip install -r setup_requirements.txt
       - name: Build and test with tox
         run: tox -e ${{ matrix.python-version.tox }}
+      - name: Publish coverage 
+        uses: orgoro/coverage@v3.1
+        with:
+            coverageFile: coverage-py3.11.xml
```

Comment:
I ran this locally with `tox -e py311` which I think this matrix will do, and it looks like this file is named `coverage-py311.xml` (without the `.` in `3.11`)

Reply:
Ah, yeah the action fails with this error too:
```
Error: "could not read file coverage-py3.11.xml"
```

Reply:
Thanks, nice catch - fixed that now. 
I was invoking locally `tox -e py3.11` which also works but it's not what's in the github action.